mantidproject/mantid
# 14955 stretched exp ft origin

Fixes #14955 

To test:
- Review code changes
- Load the files from the Usage Example of the Documentation into MantidPlot, and setup the appropriate model in the Fit-Window widget. Then, change the "origin" parameter and then Plot-Guess to check that the curve shifts along the X-axis.

Release notes in [here](http://www.mantidproject.org/ReleaseNotes_3_6_Framework_Changes#Improved_2)
